translator/totomlconfig/sampleConfig/prometheus_config_windows.json

```diff
@@ -6,6 +6,7 @@
     "metrics_collected": {
       "prometheus": {
         "cluster_name": "TestCluster",
+        "drop_unsupported_metric": false,
```

Comment:
Is there any new test config with this value set to `true`? 

Furthermore, does the value in the JSON have to exist - is this a non-breaking change in the schema config?

Reply:
No this param does not need to be existed since the default value for [this param has been set to true](https://github.com/aws/amazon-cloudwatch-agent/pull/407/files#diff-8cf20937efbe2691d764eb3fdee2185782480ef35e9c4f566f1180d89324698fR9), which is the usual behavior for CWAgent when filtering Prometheus's metrics.